.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,68 @@
+# For most projects, this workflow file will not need changing; you simply need
```

Comment:
Looks like there's quite a few complaints about some go code.

Reply:
I marked all of these as WontFix, since it's meant to run in a trusted context.